flink-doris-connector/src/main/java/org/apache/doris/flink/tools/cdc/DatabaseSync.java

```diff
@@ -235,11 +243,36 @@ protected boolean isSyncNeeded(String tableName) {
         LOG.debug("table {} is synchronized? {}", tableName, sync);
         return sync;
     }
+    /**
+     * Filter table that many tables merge to one
+     */
+    protected HashMap<Pattern,String> multiToOneRulesParser(String multiToOneOrigin,String multiToOneTarget){
+        if(StringUtils.isEmpty(multiToOneOrigin) || StringUtils.isEmpty(multiToOneTarget)){
```

Comment:
Here I have a question, should i use this method to check it, because it will import a new package, I do not know whether it has some bad impact

Reply:
Are you referring to the stringutils utility class? If yes, you can use flink’s own stringutils

Reply:
thanks,let me adjust it